<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyPSWDialog</class>
 <widget class="QDialog" name="ModifyPSWDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>140</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>140</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>140</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>修改密码</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="Label">
       <property name="text">
        <string>旧密码：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="LineEdit_OldPSW">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;输入完成后按回车键&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="Label_2">
       <property name="text">
        <string>新密码：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="LineEdit_NewPSW"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="Label_3">
       <property name="text">
        <string>确认密码：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="LineEdit_DoubleNewPSW"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_OK">
       <property name="text">
        <string>确定</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Cancel">
       <property name="text">
        <string>取消</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
